project/Dependencies.scala

```diff
@@ -29,6 +29,8 @@ object Dependencies {
   val akkaTestKit: ModuleID =       "com.typesafe.akka"   %%  "akka-testkit"         % akkaV     % "test"
   val akkaHttpTestKit: ModuleID =   "com.typesafe.akka"   %%  "akka-http-testkit"    % akkaHttpV % "test"
 
+  val cromwellClient: ModuleID =    "org.broadinstitute.cromwell" %% "cromwell-client" % "0.1-15a30475d-SNAP"
+  
```

Comment:
is there a non-SNAP version to use?

Reply:
no - cromwell doesn't currently automatically build and push to artifactory yet

Reply:
gotcha, thanks